docs/source/reference/debug.rst

```diff
@@ -34,13 +34,5 @@ You can check if debug mode is enabled with :func:`chainer.is_debug` function.
 Deprecated interface
```

Comment:
I think the section itself can be removed.

Reply:
This section is a list of "Deprecated interface", and there's nothing to list as `DebugMode` has been removed.
So I think it's OK to remove the section.